pkg/controller/useraccount/useraccount_controller_test.go

```diff
@@ -749,6 +777,65 @@ func TestReconcile(t *testing.T) {
 		require.NoError(t, err)
 		require.True(t, util.HasFinalizer(userAcc, toolchainv1alpha1.FinalizerName))
 	})
+	// delete Che user fails
+	t.Run("delete che user fails", func(t *testing.T) {
+		// given
+
+		// when the member operator secret exists and has a che admin user configured then che user deletion is enabled
+		restore := test.SetEnvVarsAndRestore(t,
+			test.Env("WATCH_NAMESPACE", "toolchain-member"),
+			test.Env("MEMBER_OPERATOR_SECRET_NAME", "test-secret"),
+			// Che has two different routes for Che & Keycloak but CRW uses a single route for both.
+			// For tests we'll assume they're two different routes.
+			test.Env(configuration.MemberEnvPrefix+"_CHE_KEYCLOAK_ROUTE_NAME", "keycloak"),
+		)
+		defer restore()
+
+		defer gock.OffAll()
+		gockTokenSuccess()
+		gockFindUserNoBody("johnsmith", 400)
+
+		memberOperatorSecret := newSecretWithCheAdminCreds()
+		userAcc := newUserAccount(username, userID, false)
+		r, req, fakeClient := prepareReconcile(t, username, userAcc, preexistingUser, preexistingIdentity, memberOperatorSecret, cheRoute(true), keycloackRoute(true))
+
+		// when
+		res, err := r.Reconcile(req)
+		require.NoError(t, err)
+		assert.Equal(t, reconcile.Result{}, res)
+
+		// then
+		userAcc = &toolchainv1alpha1.UserAccount{}
+		err = r.client.Get(context.TODO(), types.NamespacedName{Name: username, Namespace: test.MemberOperatorNs}, userAcc)
+		require.NoError(t, err)
+
+		// Check that the finalizer is present
+		require.True(t, util.HasFinalizer(userAcc, toolchainv1alpha1.FinalizerName))
+
+		// Set the deletionTimestamp
+		userAcc.DeletionTimestamp = &metav1.Time{time.Now()} //nolint: govet
```

Comment:
maybe you can use `metav1.Now()` to avoid having to comment with `//nolint: govet`?

Reply:
It seems it's not that straightforward, `userAcc.DeletionTimestamp` expects a pointer and metav1.Now() just returns a value.

Reply:
yeah, indeed... ok then, let's keep it like that if you prefer